chore: adjust technical user role documentation

Phil91 opened the pull request:
Description

Adjusted the upgrade script for the keycloak roles

Why

Currently the role assignment is missing one role, this is added with this pr

Issue

N/A

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [x] I have created and linked IP issues or requested their creation by a committer
- [x] I have added copyright and license headers, footers (for .md files) or files (for images)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have added tests that prove my changes work
- [x] I have checked that new and existing tests pass locally with my changes
- [x] I have commented my code, particularly in hard-to-understand areas

Phil91 commented:
this is the script we should execute on our environments:

```sql
WITH users AS (
    SELECT DISTINCT identity_id
    FROM portal.identity_assigned_roles AS ia
        JOIN portal.user_roles AS ur
        ON ia.user_role_id = ur.id
    WHERE ur.user_role = 'Dataspace Discovery'
),
roles_to_insert AS (
    SELECT DISTINCT atu.identity_id, ur.id AS user_role_id
    FROM users AS atu
    CROSS JOIN (
        SELECT id
        FROM portal.user_roles
        WHERE user_role = 'Identity Wallet Management'
    ) AS ur
)
INSERT INTO portal.identity_assigned_roles (identity_id, user_role_id)
SELECT rt.identity_id, rt.user_role_id
FROM roles_to_insert AS ( AS rt
    LEFT JOIN portal.identity_assigned_roles AS iar
    ON rt.identity_id = iar.identity_id AND rt.user_role_id = iar.user_role_id
WHERE iar.identity_id IS NULL;

```

evegufy reviewed (changes requested):
@Phil91 The initial description [here](https://github.com/eclipse-tractusx/portal-assets/blob/6e11c1ac2e48ad22513fd0eab88eb99465a1cadc/developer/Technical%20Documentation/Version%20Upgrade/portal-upgrade-details.md#technical-role---update) should also get updated for adding Identity Wallet Management to Connector.

And are you certain that you removed entirely the assignment of Identity Wallet Management to ex- App Tech User users, could you please check again?

Phil91 commented:
> @Phil91 The initial description [here](https://github.com/eclipse-tractusx/portal-assets/blob/6e11c1ac2e48ad22513fd0eab88eb99465a1cadc/developer/Technical%20Documentation/Version%20Upgrade/portal-upgrade-details.md#technical-role---update) should also get updated for adding Identity Wallet Management to Connector.
> 
> And are you certain that you removed entirely the assignment of Identity Wallet Management to ex- App Tech User users, could you please check again?

i've adjusted the initial description.
There was one assignment of Identity Wallet Management to App Tech User left as well, this is now removed.
